frontend/public/components/pod-logs.jsx

```diff
@@ -5,59 +5,70 @@ import { Dropdown, LoadingInline, ResourceLog, ResourceName } from './utils';
 
 // Component to container dropdown or conatiner name if only one container in pod.
 const ContainerDropdown = ({currentContainer, containers, kind, onChange}) => {
-  const resourceName = (container) => <ResourceName name={container.name || <LoadingInline />} kind={kind} />;
+  const resourceName = (container) => {
+    const name = container.name || <LoadingInline/>;
```

Comment:
Not new, but I'd expect if `container` is defined `container.name` would be, too, since name is required by the API. Do we default container to `{}` somewhere?

Reply:
Good question, I'll look over it and see if I can just remove this default.